docs/public-networks/get-started/connect/sync-node.md

```diff
@@ -113,6 +113,13 @@ Fast sync is the default for named networks specified using the [`--network`](..
 
 Using fast sync with [private transactions](../../../private-networks/concepts/privacy/index.md) isn't supported.
 
+:::note Sync nodes for BFT
+
+If you're using a Byzantine Fault Tolerance (BFT) consensus mechanism such as QBFT or IBFT 2.0, set your node's [synchronization mode](/public-networks/get-started/connect/sync-node#run-a-full-node) to either `FAST` or `FULL`. 
+By default, the configuration is set to `--sync-mode=FAST`.
+
+:::
+
```

Comment:
```suggestion
```

This note is kind of randomly placed. As in my previous comment, I recommend instead adding a note at the top of the [Run a full node section](https://besu.hyperledger.org/public-networks/get-started/connect/sync-node#run-a-full-node) (maybe right after the intro paragraph) stating that snap sync and checkpoint sync are not supported for QBFT and IBFT 2.0 networks.

Reply:
Hey @alexandratran. Placement at the top seems like it would be a bit out of place so early on.   I thought it made sense to put this between the fast sync section  and the next section which explains  full sync (since these are the options the user will have to consider)

Reply:
My recommendation was to add a note at the top of the "Run a full node" section where _fast_, _snap_, and _checkpoint_ sync are all mentioned. Here it would be relevant to note that _snap_ and _checkpoint_ sync are NOT supported for private networks.

This would make sense for users looking into running a full node via snap or checkpoint on a private network – they would now be aware that they cannot do that. And if they go straight to the fast or full sync sections, and don't see a warning, it doesn't make a difference for them because those options are supported anyways.

Does that make sense? We can discuss face to face on Monday if needed, but we shouldn't spend too much time on this.

Reply:
:::note Sync nodes for BFT

If you're running a node on a [QBFT](../../../private-networks/how-to/configure/consensus/qbft.md) or [IBFT 2.0](../../../private-networks/how-to/configure/consensus/ibft.md) network, your node must use [fast sync](#fast-synchronization) or [full sync](#run-an-archive-node). 

Full sync is set by default.

:::